<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>901</width>
      <height>641</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Устройства</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>561</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Комната</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Устройство</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_refresh">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>511</width>
        <height>561</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Устройство</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Пользователь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Врмея начала</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время окончания</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Добавить настройку</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_AddSettings">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>550</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить настройку</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_device">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_start">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>118</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_end">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>190</y>
        <width>118</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label1_time">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Время начала</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_time">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Время окончания</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>260</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Команда для выполнения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2_time_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Устройство</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
